.rubocop.yml

```diff
@@ -1,4 +1,7 @@
+inherit_from: .rubocop_todo.yml
+
 AllCops:
+  NewCops: enable
```

Comment:
I prefer to have cops enumerated and explicitly enabled.

Reply:
Removed.